### PR TITLE
example added

### PR DESCRIPTION
**Pull request by vajratools**
I struggled to find a way to implement the lib for my usecase. So I made a simple examle which creates dynamic svg from variable and saves in into pdf. Hope this will help to someone else like me.

### CODE REVIEW

**Comment by vajratools** [comment]
This is my first contribution in foss, please let me know why this change is not merged yet into the main repo?

File: examles/example1.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  </head>
+  <body>
+    <button id="downloadpdf">Download PDF</button>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.0.0/jspdf.umd.min.js" integrity="sha512-g77bZKU4ktH2I5nNioWzOMcbd3fN/svB0vQM73Uo5GRn/EGfcSJB0DlR1ithxnFsDaa0HmlOwHYiUFeM1P3RRw==" crossorigin="anonymous"></script>
+    <script src="../dist/svg2pdf.umd.min.js"></script>
+
+    <script>
+      const { jsPDF } = window.jspdf
+
+      var svgText = '<svg width="100" height="100"><rect x="0" y="0" width="200" height="100" fill="green"></rect></svg>'
```
Comment (yGuy) [inline examles/example1.html:15]: That's a weird SVG, only showing a rectangle that is larger than the viewport. Couldn't we use something more beautiful? This is test-case image quality.

**Comment by yGuy** [comment]
The demo is really super barebones and doesn't seem that helpful, IMHO. It only shows a single API functionality with some magic numbers no one will understand. 
The code is also quite inconsistent, using different loading techniques (CDN, local dependencies) and inconsistent coding (const + var). 

I fail to see the value of this addition. Even with improved code quality and a nicer user experience at "runtime".

File: examles/example1.html (hunk 1 above)
Comment (vajratools) [reply examles/example1.html:15]: Sure, I will modify the svg image. My idea was to give an example, How to use svg2pdf lib to allow user to download an image which generated sricpt?

**Pull request closed without merging**